generic/piranha/test-resources/java/feature_flag_system_1/control/input/XPFlagCleanerPositiveCases.java

```diff
@@ -33,6 +33,10 @@ enum AnotherTestExperimentName {
     SOME_OTHER_FLAG
   }
 
```

Comment:
can we add a test case where the input is 
```
enum TestEmptyEnum {
}
```
Essentially, to avoid deleting enums that are empty to begin with.

Reply:
Done